properties.schema

```diff
@@ -122,7 +139,7 @@
       "title": "Hot Graphic Pins",
       "items": {
         "type": "object",
-	      "inputType": "Hotgraphic:pins",
+          "inputType": "Hotgraphic:pins",
```

Comment:
Minor: indentation.

Reply:
I added it in like that because it was indented before and thought it was to indicate something we added 😄 

Will sort that now.